articles/tutorials/how-to-update-applications-client-secret.md

```diff
@@ -16,11 +16,11 @@ Once you have changed your application's `client_secret` you will need to update
 
 ## How to generate a new client_secret
```

Comment:
There is also a change here, the PATCH secret endpoint works as usual, you can send the 'client_secret' parameter and update your secret (encoded if it was encoded, not encoded if not). 

But also, there is a new endpoint `POST /api/v2/clients/{id}/rotate-secret`, that would rotate the secrets (non encoded secrets) with a secret we are going to generate ourselves. This is the recommended approach, since we ensure a safe secret.

Keep in mind if the customer has an old client with encoded secrets, they may want to use this old approach to keep using encoded secrets.